sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
```

Comment:
Here I have to use str to convert integer to string and then compare with status.